starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,81 @@
 // Names and Input
+const hacker1 = 'Eduardo';
+const hacker2 = 'Candido';
+let bonus = 'a man, a plan a canal panama';
+let lorem =
+	'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nec aliquam erat. Ut porta nibh ornare metus dignissim lobortis ac sed nunc. Morbi eget est euismod, accumsan urna id, eleifend nunc. Mauris quis elit sit amet odio interdum luctus quis sit amet diam. Quisque ultricies ligula non justo imperdiet laoreet. Nunc lobortis tincidunt pellentesque. Vestibulum rutrum urna vitae nisl mattis facilisis et eu urna. Nam finibus ut urna eget vehicula. Vestibulum et ex lacus. Donec interdum augue augue, at convallis arcu pellentesque vitae. Morbi posuere ornare erat id venenatis. Curabitur mollis nunc dolor, vitae lacinia dolor tristique vitae. Aenean ac purus hendrerit, porta quam ut, ornare ligula. Aliquam erat volutpat. In malesuada sapien molestie turpis faucibus, eget interdum mi scelerisque. Fusce quis luctus mauris. Nam dapibus vel felis vel finibus. Vestibulum venenatis mollis nisi, nec ornare neque. In quis orci sem. Nunc molestie in eros vitae blandit. Sed tempor sollicitudin faucibus. Praesent dui dolor, molestie eget ante ut, efficitur varius ex. Proin vitae tristique odio. Praesent efficitur scelerisque viverra. In hac habitasse platea dictumst. In ultricies blandit augue, nec aliquet felis fringilla quis. Aliquam blandit massa ac massa rutrum varius. Sed nec odio ac urna tincidunt consectetur ut dignissim felis. Donec et pulvinar sapien. Vestibulum ac aliquam nisi. Sed eu placerat risus, non ullamcorper enim. Fusce blandit egestas mauris, ut porttitor turpis. Cras bibendum arcu tempor, pulvinar enim non, malesuada eros. Curabitur eget odio in dolor dignissim pellentesque.';
```

Comment:
Just a reminder, you can use backticks `` to put linebreaks within a string. This will let you use all 3 paragraphs instead of just 1.